modules/core/src/test/java/org/apache/ignite/internal/managers/checkpoint/GridCheckpointManagerAbstractSelfTest.java

```diff
@@ -751,13 +711,13 @@ private boolean checkCheckpointManager(Ignite g) throws Exception {
     }
 
     /**
-     * Wrapper around {@link GridTestUtils#retryAssert(org.apache.ignite.IgniteLogger, int, long, GridAbsClosure)}.
+     * Wrapper around {@link GridTestUtils#waitForCondition(org.apache.ignite.internal.util.lang.GridAbsPredicate, long).
      * For the given closure provides count of retries, configured by {@link #retries} attribute.
      * @param assertion Closure with assertion inside.
```

Comment:
Method semantic has been changed.
Now "assertion" param is a closure with condition and no assertion is expected to be thrown from the closure.

Reply:
Let's get rid of this method either and rewrite places where it is used with waitForCondition.